lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -1650,7 +1650,6 @@ sub alignment_bam_file_paths {
 # The calling process is responsible for cleaning up the allocation after we are done with it.
 sub revivified_alignment_bam_file_paths {
```

Comment:
I know this didn't change in this PR but why is this plural? It only returns on path.